src/server.ts

```diff
@@ -1,25 +1,21 @@
 import compression from 'compression'
 import cors from 'cors'
-import dotenv from 'dotenv'
 import express from 'express'
 import { createServer } from 'http'
 import path from 'path'
-
-// Load the env vars before initializing code that depends on them
-if (process.env.NODE_ENV !== 'production') {
-  dotenv.config()
-}
-
-// Initialize the models
-import './sequelize'
-
-import findOrCreateProfile from './findOrCreateProfile'
+// The autoformatter alphabetizes imports and moves all imports before other code in the file
+import './00_loadEnv'
```

Comment:
stupid problems call for stupid solutions

Reply:
This is the second time this formatter has caused some annoyance. It better start behaving >:(